include/openenclave/internal/report.h

```diff
@@ -80,6 +80,13 @@ OE_STATIC_ASSERT(
     }
 #define X509_OID_FOR_QUOTE_STRING "1.2.840.113556.10.1.1"
 
+// ISO(1).ANSI(2).USA(840).Microsoft(113556).ACC(10).Classes(1).Subclass(2)
+#define X509_OID_FOR_EVIDENCE_EXT                            \
```

Comment:
My understanding is that evidence defines/typedefs go in evidence.h
Whereas report.h is for legacy report defines/typedefs

In addition, I think the define needs "OE" or "OPENENCLAVE" in the name somewhere since you use this to indicate that the evidence has an OE-specific header in it (that holds the UUID).   If standard OIDs are defined in the future for standard evidence formats, this name would be misleading.